### ISSUE TITLE
[Bug]: user set model slugs are lost if saving a record

### ISSUE THREAD

**Issue by ivan-kocienski-gfsc** [geeksforsocialchange/PlaceCal#2358]
## Description

We use slugs on Partners, Tags and Sites.

We use a gem called [FriendlyId](https://github.com/norman/friendly_id).

The problem is when editing a model, if the user sets a custom slug value and (with invalid values elsewhere) saves then the they are taken back to the form but the user provided slug value has not been persisted.

**Comment by katjam** [comment]
I think I saw somewhere that @kimadactyl suggests not letting folks edit the slugs.

**Comment by kimadactyl** [comment]
Yes - i think the real problem here is that rach shouldn't be a root user
